src/Network/Network/ChangeLog.md

```diff
@@ -36,6 +38,14 @@
     - `Update-AzVirtualHubBgpConnection`
     - `Remove-AzVirtualHubBgpConnection`
 * Onboard Azure HDInsight to Private Link Common Cmdlets
+* Support for new Match Variables in WAF Exclusions
+* Onboard Virtual Network Encryption to Virtual Network Cmdlets
+* Added new cmdlets to support Per Rule Exclusions for Application Gateway WAF
+    - `New-AzApplicationGatewayFirewallPolicyExclusionManagedRuleSet`
+    - `New-AzApplicationGatewayFirewallPolicyExclusionManagedRuleGroup`
+    - `New-AzApplicationGatewayFirewallPolicyExclusionManagedRule`
+    - Also updated cmdlet to add the property for configuring ExclusionManagedRuleSet within Exclusions
+        - `New-AzApplicationGatewayFirewallPolicyExclusion`
```

Comment:
Please move this part to the "Upcoming Release" sesion

Reply:
@isra-fel  finally done. sorry, didn't loook much at the details...